base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/editing/iocs/dialog/EditPanel.java

```diff
@@ -138,34 +142,34 @@ public EditPanel(Composite parent, int style, MessageDisplayer dialog) {
     /**
      * Sets the IOC view model used by the panel.
      * 
-     * @param viewModel
+     * @param editableIoc
      *            The view model.
      */
-    public void setViewModel(final IocViewModel viewModel) {
+    public void setViewModel(final EditableIoc editableIoc) {
 
-        macros.setViewModel(viewModel);
-        pvVals.setViewModel(viewModel);
-        pvSets.setViewModel(viewModel);
+        macros.setViewModel(editableIoc);
+        pvVals.setViewModel(editableIoc);
+        pvSets.setViewModel(editableIoc);
 
-        bind(viewModel);
+        bind(editableIoc);
     }
 
     /**
      * Binds view model values to widgets.
      * 
-     * @param viewModel
+     * @param editableIoc
      *            The IOC view model.
      */
-    private void bind(IocViewModel viewModel) {
+    private void bind(EditableIoc editableIoc) {
         DataBindingContext bindingContext = new DataBindingContext();
         bindingContext.bindValue(WidgetProperties.text(SWT.Modify).observe(selectedIoc),
-                BeanProperties.value("name").observe(viewModel));
+                BeanProperties.value("name").observe(editableIoc));
```

Comment:
This throws some exceptions as the EditableIoc is missing a setName method. I understand this is by design and doesn't impact functionality here, however it does clutter up the log a bit. Could you somehow handle/get rid of these exceptions?